src/components/InfoCard/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from "react";
+import { Stack, Typography, useTheme, Box } from "@mui/material";
+import { CircledInfoCard, InfoCardWrapper, ResultNumber } from "../../styles";
+
+interface IInfoCard {
+  isBig?: boolean;
+  bgColor?: string;
+  number: string | number;
+  iconColor: string;
+  icon: any;
+  title: string;
+}
+
+export const InfoCard: FC<IInfoCard> = ({
+  number,
+  isBig = false,
```

Comment:
Better to change `isBig` (boolean) props to `size` (string) with the next variants: 'sm" (small) - default value, "lg" (large).